### PR TITLE
Epic 7 - Yurii Steshenko

### PR DESCRIPTION
**Pull request by SeT002**

### CODE REVIEW

**Comment by okohuch** [comment]
стоп

**Comment by izagor** [comment]
+ 12-19-2023

File: ai_12/yurii_steshenko/Epic_7/live_coding_task.cpp [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+  // Розрахунок довжини банера
+  length = width * surname.length();
+  cout << length<<endl;
+  // Виведення довжини банера
+  cout << "The length of the banner: " << length << " meters" << endl;
+
+  // Визначення символів, які будуть знаходитись на 3-х центральних(по довжині) метрах поля
+  // 1) при написанні прізвища з лівого краю поля —>  до правого
+  int start_index = (int) (length / 2) - 2 * width;
+  int end_index = (int) (length / 2) + 2 * width;
```
Comment (fm2lpnu) [inline ai_12/yurii_steshenko/Epic_7/live_coding_task.cpp:77]: Mismatch between code and comment.
The length had to be the field length, not banner

File: ai_12/yurii_steshenko/Epic_7/vns_practice_4.cpp [hunk 2]
```diff
@@ -0,0 +1,11 @@
+#include <iostream>
+#include <cmath>
+using namespace std;
+
+int main() {
+    for (int i = 0; i <= 10; i++) {
+        cout << 2 << "^" << i << " = " << pow(2, i) << endl;
```
Comment (fm2lpnu) [inline ai_12/yurii_steshenko/Epic_7/vns_practice_4.cpp:7]: Incorrect implementation, you implemented power of two (https://en.wikipedia.org/wiki/Power_of_two).
Task requires implementation of factorial(https://en.wikipedia.org/wiki/Factorial).
From math point factorial looks like `5! = 5*4*3*2*1`.

**Review by fm2lpnu — APPROVED** [review]

**Pull request merged**